packages/EasyApiPlatform/src/Bridge/Symfony/DependencyInjection/EasyApiPlatformExtension.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyApiPlatform\Bridge\Symfony\DependencyInjection;
+
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\Extension;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
+use Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
+
+final class EasyApiPlatformExtension extends Extension implements PrependExtensionInterface
+{
+    /**
+     * @param mixed[] $configs
+     *
+     * @throws \Exception
+     */
+    public function load(array $configs, ContainerBuilder $container): void
+    {
+        $this->processConfiguration(new Configuration(), $configs);
+
+        $loader = new PhpFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
+        $loader->load('services.php');
+    }
+
+    public function prepend(ContainerBuilder $container): void
```

Comment:
Then let's remove this method and not implement the PrependExtensionInterface.

Reply:
Done.